sophiread/CMakeLists.txt

```diff
@@ -31,17 +31,17 @@ file(COPY ${CMAKE_SOURCE_DIR}/resources/data DESTINATION ${CMAKE_BINARY_DIR})
 add_compile_options(
   -O3
   -std=c++20
-  -march=native
+#  -march=native
   -ffast-math
   -pthread
   -Wall
   -Wextra
-  # ${OpenMP_CXX_FLAGS}
```

Comment:
similar as above, since we are using `tbb`, there is no need to link against openMP anymore.